kfdefs/base/jupyterhub/notebook-images/ope.yaml

```diff
@@ -0,0 +1,30 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: ope
+  labels:
+    component.opendatahub.io/name: jupyterhub
+    opendatahub.io/component: 'true'
+    opendatahub.io/notebook-image: 'true'
+  annotations:
+    opendatahub.io/notebook-image-name:
+      "Open Education (OPE)"
+    opendatahub.io/notebook-image-url:
+      "https://github.com/jappavoo/UndertheCovers"
+    opendatahub.io/notebook-image-desc:
+      "Template Image for Open Education (OPE)"
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+    - annotations:
+        openshift.io/imported-from: quay.io/ope/ope_book
+      name: stable-opf
+      from:
+        kind: DockerImage
+        name: 'quay.io/ope/ope_book:stable-opf'
```

Comment:
This image does not exist, what you are looking for is: `quay.io/opeffort/ope_book:stable-opf`

Reply:
@jappavoo can you please confirm this image is existing , i cant access it either.

Reply:
Sorry.  I have fixed the urls.  Too much cut-n-pasting and not enough coffee.  Thanks for your careful checking.